### ISSUE TITLE
Programmers and DevOps should have general knowledge of markdown

### ISSUE THREAD

**Issue by dkbaldwin** [inDevelopme/indevelopme-coaching-publications#70]

**Comment by jasalise** [comment]
Markdown is a lightweight markup language with plain-text formatting syntax designed to be easy to read and write. It allows you to format text using simple, intuitive symbols. Markdown is commonly used for formatting documentation, readme files on GitHub, forum posts, and other types of writing intended for the web.

Markdown syntax includes features like headers, emphasis (bold and italic text), lists (both ordered and unordered), links, images, code blocks, and more. It is often preferred for its simplicity and readability compared to more complex markup languages like HTML.

Here's an example of Markdown syntax:

```
# Heading 1

## Heading 2

### Heading 3

*Italic Text*

**Bold Text**

- Unordered List Item 1
- Unordered List Item 2

1. Ordered List Item 1
2. Ordered List Item 2

[Link Text](https://example.com)

![Image Alt Text](image.jpg)

`Inline Code`

```
Code Block
```

```python
# Syntax Highlighted Code Block
print("Hello, Markdown!")
```

Markdown is converted into HTML when rendered in web browsers, making it a convenient way to write content for the web without directly using HTML tags.

**Comment by jasalise** [comment]
**[Resources ](https://www.markdownguide.org/getting-started/)**

**Comment by jasalise** [comment]
Essay should have:
Blockqutoes
list
links to external resources 
code / code block
emphasis (bold, italic, both)
headings
paragraphs

**Comment by jasalise** [comment]
# Popcorn: A Popping Phenomenon

Introduction:
Popcorn, a snack cherished for its crunchiness and flavor, has a rich history and cultural significance that stretches back thousands of years. From its humble beginnings as a staple of indigenous American diets to its current status as a movie theater staple and gourmet treat, popcorn has captivated people's imaginations and taste buds across the globe. In this essay, we will explore the fascinating journey of popcorn, delving into its historical roots, scientific principles, cultural significance, and modern-day appeal.

Historical Roots:
Popcorn's origins can be traced back to ancient Mesoamerican civilizations, where indigenous peoples cultivated a special variety of corn known as Zea mays everta. According to archaeologists, popcorn was consumed as early as 3600 BCE in what is now present-day Peru. As Dr. Bruce Smith, curator at the Smithsonian Institution, states:

>
>"Popcorn was a very important part of early human diet in the Americas."
>

The Aztecs and other indigenous groups not only ate popcorn as a snack but also used it in religious ceremonies and decorative ornaments. When European explorers arrived in the Americas, they encountered popcorn and introduced it to the rest of the world.

Popcorn's Rise to Popularity:
In the 19th century, popcorn gained widespread popularity in the United States, particularly at fairs, circuses, and other public events. Street vendors would pop corn kernels in large kettles over open flames, enticing crowds with the aroma of freshly popped corn. According to Andrew F. Smith, author of "Popped Culture: A Social History of Popcorn in America":

>
>"Popcorn became popular at fairs and circuses in the middle of the 19th century. It was an inexpensive luxury that became widely available."
>

The Great Depression further solidified popcorn's status as a beloved snack, as movie theaters began selling it as an affordable treat to attract patrons during tough economic times.

Scientific Principles:
The process of popcorn popping is a fascinating example of physics in action. Each popcorn kernel contains a small amount of moisture trapped inside a hard outer shell. When heated, the moisture turns into steam, building pressure within the kernel until it explodes. As explained by food scientist Dr. Joe Vinson:

>
>"When the steam pressure reaches around 135 pounds per square inch, it ruptures the pericarp or shell of the popcorn kernel, causing it to explode and turn inside out, effectively turning the kernel inside out and exposing the fluffy starchy interior."
>

This sudden release of pressure transforms the kernel into the fluffy, edible snack we know as popcorn.

Cultural Significance:
Popcorn holds cultural significance in many societies around the world. In the United States, it is closely associated with movie theaters, where it has become an integral part of the cinematic experience. As cinema historian Katherine Nichols observes:

>
>"Popcorn and movies have been a perfect pairing for nearly a century. The aroma of freshly popped popcorn creates a sensory experience that enhances the movie-watching experience."
>

In addition to its association with entertainment, popcorn is also enjoyed during holidays, sporting events, and social gatherings, symbolizing togetherness and celebration.

Conclusion:
In conclusion, popcorn's journey from ancient snack to modern-day favorite is a testament to its enduring appeal and cultural significance. Its historical roots, scientific principles, and widespread popularity make it a truly remarkable culinary phenomenon. Whether enjoyed at the movies, at home, or at special events, popcorn continues to delight and captivate people of all ages, proving that some snacks truly stand the test of time.

**Comment by justaskdavidb** [comment]
Looking great. Can't wait to hear what you learned.
